examples/crm/src/notes/Note.tsx

```diff
@@ -163,25 +148,36 @@ export const Note = ({
                     <Box flex={1}>
                         {note.text
                             .split('\n')
-                            .map((paragraph: string, index: number) => (
-                                <Box
-                                    component="p"
-                                    fontFamily="fontFamily"
-                                    fontSize="body1.fontSize"
-                                    lineHeight={1.3}
-                                    marginBottom={2.4}
-                                    key={index}
-                                >
-                                    {paragraph}
-                                </Box>
-                            ))}
+                            .map(
+                                (
+                                    paragraph: string,
+                                    index: number,
+                                    array: string[]
+                                ) => (
+                                    <Box
+                                        component="p"
+                                        fontFamily="fontFamily"
+                                        fontSize="body1.fontSize"
+                                        lineHeight={1.3}
+                                        marginTop={index === 0 ? 0 : undefined}
+                                        marginBottom={
```

Comment:
**non-blocking(nit-pick):** can this be replaced with a [:last-of-type](https://developer.mozilla.org/fr/docs/Web/CSS/:last-of-type) on parent's `sx` ?